lucid/optvis/render.py

```diff
@@ -250,11 +250,11 @@ def make_optimizer(optimizer, args):
            "optimizer, or tf.train.Optimizer instance.")
 
 
-def import_model(model, t_image, t_image_raw=None, scope="import"):
+def import_model(model, t_image, t_image_raw=None, scope="import", input_map=None):
```

Comment:
Doesn't need to be addressed in this PR, but it's pretty weird that we rely on this helper function so much, and that it lives in render. It might be worth improving  `model.import_graph()` so that it can also return `T` and then deprecating this.)